Mathlib/Order/Bounds/Image.lean

```diff
@@ -196,6 +196,45 @@ theorem map_isLeast : IsLeast s a → IsGreatest (f '' s) (f a) :=
 
 end Antitone
 
+section StrictMono
+
+variable [LinearOrder α] [Preorder β] {f : α → β} {a : α} {s : Set α}
+
+lemma StrictMono.mem_upperBounds_image (hf : StrictMono f) :
+    f a ∈ upperBounds (f '' s) ↔ a ∈ upperBounds s := by simp [upperBounds, hf.le_iff_le]
+
+lemma StrictMono.mem_lowerBounds_image (hf : StrictMono f) :
+    f a ∈ lowerBounds (f '' s) ↔ a ∈ lowerBounds s :=  by simp [lowerBounds, hf.le_iff_le]
+
+lemma StrictMono.map_isLeast (hf : StrictMono f) : IsLeast (f '' s) (f a) ↔ IsLeast s a := by
```

Comment:
I am a bit agnostic here, but the existing lemmas point to `map_` rather than `_image`, eg [`Monotone.map_isLeast`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Bounds/Image.html#Monotone.map_isLeast)

Reply:
Ok, let's keep the names as is.

bors merge